specs-go/config.go

```diff
@@ -3,21 +3,22 @@ package specs
 import "os"
 
 // CurrentVersion is the current version of the Spec.
-const CurrentVersion = "0.5.0"
+const CurrentVersion = "0.6.0"
 
 // Spec is the base configuration for CDI
 type Spec struct {
-	Version string `json:"cdiVersion"`
-	Kind    string `json:"kind"`
-
-	Devices        []Device       `json:"devices"`
-	ContainerEdits ContainerEdits `json:"containerEdits,omitempty"`
+	Version        string            `json:"cdiVersion"`
+	Kind           string            `json:"kind"`
+	Annotations    map[string]string `json:"annotations,omitempty"` // Annotations add meta information per device
```

Comment:
those are global per file, not per device.

Reply:
Does it make sense to have the docstring preceding the field, or is a trailing comment also common practice?